src/servers/src/http/opentsdb.rs

```diff
@@ -84,17 +84,19 @@ pub async fn put(
     let summary = params.contains_key("summary");
     let details = params.contains_key("details");
 
-    let data_points = parse_data_points(body).await?;
+    let data_point_requests = parse_data_points(body).await?;
+    let data_points = data_point_requests
```

Comment:
Why clone the `data_point_requests`? It seems that we could simply use it.

Reply:
`data_point_requests` is of type `Vec<DataPointRequest>` and `data_points` is `DataPoint`.
`data_points` is what is used for the executions in line 115. I kept `data_points_requests` because that is what the error handler accepts (check line 120) and I felt it will be unnecessary to write implement a `From` trait to convert `DataPoint` to `DataPointRequest` just for that.

Reply:
When the condition (`!summary && !details) are met, we can avoid cloning.